app/views/sufia/homepage/_recent_items.html.erb

```diff
@@ -0,0 +1,30 @@
+<%
+#First, put a bunch of eligible works into a bag:
+conditions =  {'read_access_group_ssim'=>'public'}
+how_many_works_in_bag = 15
+sort_by = ["system_modified_dtsi desc"]
+opts = {:rows=>how_many_works_in_bag, :sort=>sort_by}
+works_to_pick_from = GenericWork.search_with_conditions( conditions, opts)
+
+# Now, pick a few of these out of the bag at random to show.
+# Shuffle the bag again every few minutes.
+how_often_to_change=60 #seconds
+srand Time.now.to_i/how_often_to_change
+how_many_works_to_show = 6
+works_to_show = works_to_pick_from.sort_by{rand}[0..how_many_works_to_show-1]
+%>
+
+<h2 class="brand-alt-h2">Featured Items</h2>
+<div class="row recent-items-list">
+    <% works_to_show.each do |work| %>
+    <div class="recent-item">
+        <%= link_to main_app.curation_concerns_generic_work_path(work['id']) do %>
+          <%= image_tag  work['thumbnail_path_ss'], class: "recent-items-image", width: "100%"%>
+          <div class="recent-item-title-wrapper">
+          <p class="recent-item-title"><%= work['title_tesim'].first %></p>
+          </div>
+        <% end # link %>
+    </div>
+    <% end # each %>
+</div>
+
```

Comment:
Let's call this "Recently Added Items" (Or "Some of our New Items" or something), instead of "Featured Items". They aren't really intentionally "featured".